numpy/polynomial/legendre.py

```diff
@@ -1418,8 +1418,11 @@ def legfit(x, y, deg, rcond=None, full=False, w=None):
         y-coordinates of the sample points. Several data sets of sample
         points sharing the same x-coordinates can be fitted at once by
         passing in a 2D-array that contains one dataset per column.
-    deg : int
-        Degree of the fitting polynomial
+    deg : int or array_like
```

Comment:
Should probably separate the new bits out so that the new bits can be annotated `.. versionadded:: 1.11.0`. Not sure what the best way to do that is.